### PR TITLE
Lien Verification

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#14849

*Description of changes:*
* Includes Lien Verification at Submission
* Includes errors and alert messaging on Review / Confirm
* Updated civic address default

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by ChasBoucher — APPROVED** [review]
Looks good.

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -449,6 +456,17 @@ export default defineComponent({
 
       // If already in review mode, file the transfer
       if (localState.isReviewMode) {
+        // Verify no lien exists prior to submitting filing
+        const regSum = !localState.isJestRunning
```
Comment (dimak1) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue:474]: Could we check for Jest at the same time we check for isReviewMode, line 458?

**Review by dimak1 — APPROVED** [review]
Looks good, just one suggestion.

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:474]: We could* although that would prevent all the interactions within this block, inside our unit tests. 
Example being, triggering the error state for required fields which in turn displays validation messaging etc.

**Review by doug-lovett — APPROVED** [review]
LGTM.

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:474]: Sounds good, thanks for clarifying. Resolving.

Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue:474]: No problem at all, a very valid question!

**Pull request merged**